test/helpers.py

```diff
@@ -160,7 +160,7 @@ def run_locally(self, args):
         temp = CmdlineParser._instance
         try:
             CmdlineParser._instance = None
-            run_exit_status = luigi.run(['--local-scheduler', '--no-lock'] + args)
+            run_exit_status = luigi.run(args + ['--local-scheduler', '--no-lock'])
```

Comment:
Is there a reason for the order swap here?

Reply:
So you're fine with that change? If yes, I'll also add a note to the docs on that.

Reply:
Yeah, i'm fine with it - so long as tests pass (they aren't currently)